README.md

```diff
@@ -26,7 +26,27 @@ codebuild/bin/s2n_install_test_dependencies.sh
 codebuild/bin/s2n_codebuild.sh
 ```
 
-### Have a Question?
+## Quickstart for OSX (or other platforms)
+
+If you are building on OSX, or simply don't want to execute the entire build script above, you can use build tools like Ninja.
+
+### OSX
+
+An example of building on OSX:
+
+```sh
+brew install ninja cmake
+git clone https://github.com/${YOUR_GITHUB_ACCOUNT_NAME}/s2n.git
+mkdir s2n_build
+cd s2n_build
+
+# Build with debug symbols
+cmake -GNinja -DCMAKE_BUILD_TYPE=Debug ../s2n
+ninja -j8
```

Comment:
There was a recent comment (chat?) about builds locking up machines b/c of too much parallelism- might consider just going with the default, which on my MBP says 6.

Reply:
Modified to -j6